integration/test/basic/basic_test.go

```diff
@@ -27,39 +32,55 @@ func TestChartInstalled(t *testing.T) {
 		RestConfig:      f.RestConfig(),
 		TillerNamespace: "giantswarm",
 	}
-	helmClient, err := helmclient.New(c)
+
+	gsHelmClient, err := helmclient.New(c)
 	if err != nil {
 		t.Fatalf("could not create helmClient %v", err)
 	}
 
-	err = f.InstallResource("chart-operator-resource", templates.ChartOperatorResourceValues, ":stable")
+	// Test Creation
+	log.Printf("creating %q", cr)
+	err = f.InstallResource(cr, templates.ChartOperatorResourceValues, ":stable")
+	if err != nil {
+		t.Fatalf("could not install %q %v", cr, err)
+	}
+
+	err = waitForReleaseStatus(gsHelmClient, testRelease, "DEPLOYED")
+	if err != nil {
+		t.Fatalf("could not get release status of %q %v", testRelease, err)
```

Comment:
This was here before but the text is slightly misleading IMO.
The problem here is that the status did not get set to `DEPLOYED`, right?

Reply:
Yes exactly. The release has to get deployed first before we can retrieve a status. The output looks during the tests looks like this:
```
018/05/08 15:27:12 getting release status 3.36370905s: could not retrieve release content: tb-release: release not found
2018/05/08 15:27:16 getting release status 10.578945892s: could not retrieve release content: tb-release: release not found
2018/05/08 15:27:26 getting release status 6.521996113s: waiting for "DEPLOYED", current "PENDING_INSTALL"
``` 
I think it's fine, but if you find a better wording to make it clearer I am open.